yammbs/analysis.py

```diff
@@ -230,10 +233,13 @@ def get_internal_coordinates(
                 key = tuple(
                     (
                         internal_coordinate.a,
-                        internal_coordinate.b,
+                        int(internal_coordinate.b),  # geomeTRIC somehow makes this np.int32
```

Comment:
(not blocking) Should `internal_coordinate.a` also be cast to int?

Reply:
It's already an int, quite confusingly. (Either both being `int` or `np.int32`, etc. wouldn't be alarming to me, but similar attributes having mixed types is weird. And unfortunately what I see in an interpreter ... )